mkdocs.yml

```diff
@@ -61,13 +61,21 @@ markdown_extensions:
   - def_list
   - attr_list
   - md_in_html
-
+  - pymdownx.highlight:
+      anchor_linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
+  - admonition
+  - pymdownx.details
+  - pymdownx.superfences
 
 # Page tree
 nav:
   - Home: index.md
   - Getting Started:
     - Quick Start: get-started/quick-start.md
+    - Writing Tests: get-started/writing-tests.md 
```

Comment:
Is there also a `index.md` file in `docs` dir that keeps a similar index that needs updating?

Reply:
I don't think it's worth maintaining links in an index.md manually. The side nav is sufficient in the current layout. That home page (index.md) should have something a bit different to engage, both content and layout. With the latter the cards feature is a paid only.

Reply:
Okay in that case maybe just put/keep a few high-level links there in `index.md` and don't worry about it for now, i.e. making it different enough between the two so that people won't update it reactively thinking that it should be updated.